src/components/molecules/Alert/types.ts

```diff
@@ -3,4 +3,5 @@ export enum AlertSeverity {
     Info = 'info',
     Success = 'success',
     Warning = 'warning',
+    Basic = 'basic',
```

Comment:
After discussion with @benk-sf , this new variant will be used in portfolio management too.

No alert icon and no close button (the old behaviour) @as472